utils/frame/remote-externalities/src/lib.rs

```diff
@@ -428,10 +431,10 @@ impl<B: BlockT + DeserializeOwned> Builder<B> {
 		let keys_chunked: Vec<Vec<StorageKey>> =
 			keys.chunks(thread_chunk_size).map(|s| s.into()).collect::<Vec<_>>();
 		for thread_keys in keys_chunked {
-			let uri = Arc::clone(&uri);
+			let thread_client = client.clone();
```

Comment:
So the client is now shared between all threads? Is that maybe slowing done anything?

Reply:
Or can you send parallel requests from multiple threads?

Reply:
Kian introduced this make it faster i.e, to make parallel requests from multiple threads but I think `tokio::spawn` should be sufficient here i.e, use the tokio threadpool.

But I removed that a new RPC client is created for each thread that seems needless as it's already multi-threaded, so I I `Arc:ed` instead and the request/call is just done a separate task/thread.